src/components/img.ts

```diff
@@ -26,7 +26,7 @@ const Img: FC<Props> = ({ image, sizes, imageMappings, className }) =>
         sizes,
         srcSet: srcset(image.src, imageMappings),
         src: src(imageMappings, image.src, 500),
-        alt: image.alt,
+        alt: image.alt.withDefault(''),
```

Comment:
Yeah I noticed this... Interesting that it doesn't throw a type error 🤔

Reply:
the good old `[object Object]`

Reply:
Ok, it appears it's not throwing a type error because of `styledH`... If you switch to JSX it *does* throw an error.